docs/cli/commands/kcp.md

```diff
@@ -0,0 +1,40 @@
+# kcp
+Day-two operations tool for Kyma Runtimes
+
+## Synopsis
+
+The KCP CLI (Kyma Control Plane CLI) is a day-two operations tool for Kyma Runtimes, which allows you to view and manage the Runtimes in scale.
+It is possible to list and observe attributes and state of each Kyma Runtime and perform various operations on them, such as upgrading the Kyma version.
+You can find the complete list of possible operations as commands below.
+
+The CLI supports configuration file for common, global options needed for all commands. The config file will be looked up in this order:
+  - `--config {PATH}` option
+  - KCPCONFIG environment variable which contains the path
+  - $HOME/.kcp/config.yaml (default path).
+
+The configuration file is in YAML format and supports the following global options: oidc-issuer-url, oidc-client-id, oidc-client-secret, keb-api-url, kubeconfig-api-url, gardener-kubeconfig.
```

Comment:
```suggestion
The configuration file is in the YAML format and supports the following global options: `oidc-issuer-url`, `oidc-client-id`, `oidc-client-secret`, `keb-api-url`, `kubeconfig-api-url`, `gardener-kubeconfig`.
```

Reply:
Unfortunately this part is generated from CLI global options list, and I can't easily apply Markdown syntax here.